app-admin/kubelet/files/kubelet.service

```diff
@@ -1,19 +1,48 @@
 [Unit]
-Description=Kubernetes Kubelet
-Documentation=https://github.com/GoogleCloudPlatform/kubernetes
+Description=Kubelet Agent. Launches the kubelet and restarts when new versions are detected.
+Documentation=http://kubernetes.io
 
 [Service]
+# Hardcoded version of kubelet agent
+Environment=KUBELET_AGENT_VERSION=0.1.0
+
+# If left empty the kubelet version will be determined by contacting the kube-apiserver.
+Environment=KUBELET_PINNED_VERSION=
```

Comment:
I don't feel too strongly here, but, I have a slight preference for `KUBELET_PINNED_VERSION` (or anything that makes it clear your pinning) I think its a little more clear that, when set, the user is forgoing any auto-updates that are normally available. 

Normally, when someone sets a version env, they probably just assume they are just swapping an old version for one they need.